examples/src/ffi/cpp/ffi_robot.cc

```diff
@@ -51,7 +51,7 @@ class RobotServiceClient {
 int main(int argc, char *argv[]) {
 
   void *ptr = init_rust_runtime();
-  char *path = dial("<your robot's uri>", "<your robot's credentials>", false, ptr);
+  char *path = dial("<your robot's uri>", "<your robot's payload>", "<your robot's type>", false, ptr);
```

Comment:
isn't this switched from above given the signature is `type` then `payload`?

Reply:
Good catch thank you!